easybuild/easyconfigs/r/Racon/Racon-1.4.13-std_invalid_argument.patch

```diff
@@ -0,0 +1,46 @@
+Binary files racon-v1.4.13-old/.git/modules/vendor/spoa/index and racon-v1.4.13/.git/modules/vendor/spoa/index differ
```

Comment:
This first line should be removed (we don't care about this particular file), and be replaced with a short description of what this patch does or why it is needed, and where the patch comes from (upstream reference, or author)